CHANGES.md

```diff
@@ -1,3 +1,11 @@
+## 0.2.8.0
+
+ * Add `Eq1/2`, `Show1/2`, `Read1` instances with `base-4.9
+
+ * `Eq (HashSet a)` doesn't require `Hashable a` anymore, only `Eq a`.
```

Comment:
It was motivated by `Eq1` instance, as you cannot use `Hashable`, but I can leave the `Eq` instance as it is.